### PR TITLE
fix: use named version from 'vue'

### PR DESCRIPTION
**Pull request by dargmuesli**
Kind of a `chore` PR.

### 📚 Description

Resolves both current eslint warnings.
- import `version` from `Vue`
- add type to error prop in playground layout

### CODE REVIEW

**Comment by dargmuesli** [comment]
Are those changes desired / still on the radar? It's not much, just found this PR of mine still lingering around here :smile:

File: playground/layouts/error.vue [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 <script>
 export default {
-  props: ['error']
+  props: {
+    error: Error
```
Comment (danielroe) [inline playground/layouts/error.vue:10]: This prop is not always an Error.

See for example this typing in Nuxt 3:

https://github.com/nuxt/framework/blob/888bd7c145cf276bfbb93e9610ba3b6c8786007f/packages/nuxt/src/app/nuxt.ts#L81-L88

Comment (dargmuesli) [reply playground/layouts/error.vue:10]: Would it be fair to copy that typing? As it cannot be referenced, I think, as a property of an unexported interface.

Comment (danielroe) [reply playground/layouts/error.vue:10]: Yes. The source does not come from that repo but from the nitro renderer within Bridge.

Comment (danielroe) [reply playground/layouts/error.vue:10]: Wait. I just twigged this is the _playground_. Scratch that. Your original solution is fine! 😆

**Review by danielroe — APPROVED** [review]

**Pull request merged**